t/storage.t

```diff
@@ -159,26 +172,17 @@ describe "HTFeed::Storage" => sub {
       };
 
       context "when link path is the same as object path" => sub {
-        my $old_obj_dir;
-
-        before each => sub {
-          my $old_obj_dir = get_config('repository','obj_dir');
-          set_config(get_config('repository','link_dir'), 'repository','obj_dir');
-        };
-
-        after each => sub {
-          set_config($old_obj_dir,'repository','obj_dir');
-        };
-
         it "works" => sub {
           my $storage = local_storage($tmpdirs,'test','test');
+          $storage->set_storage_config(config_for_class('HTFeed::Storage::LocalPairtree')->{link_dir},'obj_dir');
```

Comment:
Again I think we want to construct a new instance of the storage with the desired config up front rather than mutating it afterwards